lib/board.rb

```diff
@@ -50,13 +50,15 @@ def render(show_ship = false)
     num_column = (@cells.keys.map {|coord| coord.delete("^0-9")}).uniq.last.to_i
     num_row = @cells.keys.map {|key| key.delete("^A-Z")}.uniq.count
     print_render = []
-    num_labels = "  " + (1 .. num_column).to_a.join(" ") + "\n"
+    num_labels = "  " + (1 .. num_column).to_a.join(" ") + " \n"
     character = "A"
+    print num_labels
     num_row.times do 
       print_render << render_helper(character,num_column,show_ship)
       character = character.next
     end
-    print num_labels + print_render.join
+    rt = num_labels + print_render.join
+    rt
```

Comment:
This was such a doozie, glad we were able to figure it out! Noted for the future to track what objects are being returned.